site/docs/components/dropdown/examples.mdx

```diff
@@ -0,0 +1,65 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="dropdown" exampleName="Default">
+
+### Dropdown
+
+Dropdown contains a button that opens a list on mouse or keyboard interaction.
+
+</LivePreview>
+
+<LivePreview componentName="dropdown" exampleName="Variants" >
+
+### Variants
+
+Dropdown has two variants: `primary` and `secondary`. The primary variant is set by default. Set to secondary with `variants=”secondary”`.
+
+Choose secondary when used in a data-dense or compact interface, or for a form with multiple columns or sections.
+
+The choice between primary and secondary variants should be considered alongside the background color it sits upon.
+
+Tip: The [Forms pattern](/salt/patterns/forms) provides further guidance on primary versus secondary control selection.
+
+</LivePreview>
+
+<LivePreview componentName="dropdown" exampleName="DefaultSelected" displayName="Preselected value" >
+
+### Preselected value
+
+By default, the dropdown’s field is empty. When the `defaultSelected` prop is passed, the dropdown field displays the pre-selected value.
+
+</LivePreview>
+
+<LivePreview componentName="dropdown" exampleName="Readonly" displayName="Read-only" >
+
+### Read-only
+
+Set Dropdown to read-only to allow values to be displayed that are predefined and not editable, for instance when a user does not have the appropriate permissions to select an option within the dropdown.
```

Comment:
nit. `Set Dropdown to read-only to allow...` reads a bit funny

Reply:
Will keep it as is for now as it's content approved :)